_posts/2019-12-09-reinvent-2019-recap.markdown

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title:  "re:Invent 2019 Recap"
+date: 2019-12-09
+tags: aws reinvent recap
+author: Nathan Glover
+image: /img/blog/reinvent-2019-recap/reinvent-opener.png
+---
+
+## Introduction
+
+{:refdef: style="text-align: center;"}
+<img src="/img/blog/reinvent-2019-recap/reinvent-opener.png" width="700">
+{: refdef}
+
+re:Invent 2019 is the largest Amazon run event that occurs yearly in Las Vegas. Since 2011 it has been progressively scaling up in size and this year I was lucky enough to be able to attend for the first time.
+
+I'm sure many readers have filled themselves in on the announcements remotely; however as someone who; in 2018 tried to digest what releases were *important*; I've decied to put together this post outlining the things that stood out to me.
```

Comment:
Many semi-colons; quite wow.

Suggest re-working this sentence. Suggest:
```
I'm sure many readers have filled themselves in on the announcements remotely. As someone who in 2018 tried to digest what releases were *important*, I've decided to put together this post outlining the things that stood out to me this year
```
Also small spelling fix on decided.